proto/private/dependencies.bzl

```diff
@@ -113,6 +121,14 @@ dependencies = {
         ],
     },
     # Dependency of `com_github_protocolbuffers_protobuf`.
+    "rules_pkg": {
+        "sha256": "a89e203d3cf264e564fcb96b6e06dd70bc0557356eb48400ce4b5d97c2c3720d",
+        "urls": [
+            "https://mirror.bazel.build/github.com/bazelbuild/rules_pkg/releases/download/0.5.1/rules_pkg-0.5.1.tar.gz",
+            "https://github.com/bazelbuild/rules_pkg/releases/download/0.5.1/rules_pkg-0.5.1.tar.gz",
+        ],
+    },
```

Comment:
This is a new dependency for com_github_protocolbuffers_protobuf?

Reply:
Yes, without it, the build would error out https://buildkite.com/bazel/rules-proto/builds/1165#f9f1919d-b4ab-4234-bdb5-1382c077d0d3